tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -576,6 +575,7 @@ export const editTextDocument = async ({
     page.locator(saveTextFileInEditorButton).click()
   ])
   await editor.close(page)
+  await page.reload()
```

Comment:
need to avoid issue with webkit

Reply:
will this cover a real bug? then we shouldn't do it

Reply:
No, it is not possible to reproduce this locally, only in CI.
this issue: 
issue after closing text editor. similar was in the #954
<img width="968" height="862" alt="Screenshot 2025-07-29 at 15 09 01" src="https://github.com/user-attachments/assets/238ffb2a-f6be-4ab0-9564-8df4113a07ae" />